app/src/main/res/layout/fragment_t_t_scheduling_screen.xml

```diff
@@ -1,15 +1,51 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".ui.main.TTSchedulingScreenFragment">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".ui.main.TTSchedulingScreenFragment"
+   >
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        >
+
+
+
+
 
-    <!-- TODO: Update blank fragment layout -->
     <TextView
+        android:id="@+id/remindi_header"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Remindi"
+        android:fontFamily="@font/poppins"
+        android:textSize="20sp"
+        android:layout_marginTop="60dp"
+        android:layout_marginLeft="18dp"
+        />
+
+
+
+    <!-- TODO: Update blank fragment layout -->
+
+
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/tt_items_recycler_view"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:text="TT Scheduling Screen"
-        android:gravity="center"/>
+        android:paddingTop="25dp"
+        android:paddingRight="16dp"
+        android:paddingBottom="50dp"
+        android:clipToPadding="false"
+        tools:listitem="@layout/tt_scheduling_item" />
+
+    </LinearLayout>
+
+
 
-</FrameLayout>
\ No newline at end of file
+</ScrollView>
```

Comment:
`RecyclerView` itself is scrollable so, remove the `ScrollView` since it serves no purpose